agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -1207,44 +1208,34 @@ func (dg *dockerGoClient) removeVolume(ctx context.Context, name string) error {
 	return nil
 }
 
-// ListPluginsWithFilters currently is a convenience method as go-dockerclient doesn't implement fitered list. When we or someone else submits
-// PR for the fix we will refactor this to pass in the fiters. See https://docs.docker.com/engine/reference/commandline/plugin_ls/#filtering.
 func (dg *dockerGoClient) ListPluginsWithFilters(ctx context.Context, enabled bool, capabilities []string, timeout time.Duration) ([]string, error) {
+	// Create filter list
+	filterList := filters.NewArgs( filters.Arg("enabled", "true"))
```

Comment:
nit: extra space, please run gofmt.
Also shouldn't the  valued read from the parameter `enabled`?